### DISCUSSION TITLE
correlation between estimated gene activity (scATAC) and RNA level in scRNA

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#1026] [Q&A] [answered]
Hi Tim,
In terms of the estimated gene activity used here https://satijalab.org/signac/articles/pbmc_vignette.html, did you have high correlation between the estimated gene activity using scATAC and RNA level in scRNA?

Thanks
Best

**Comment by timoast** [comment, marked as answer]
Hi @pengxin2019, I wouldn't necessarily expect a high correlation between gene activity for and gene expression for individual genes. You can run this on a multiome dataset to check what the approximate values are. We mainly use this for integration with scRNA-seq datasets, and in that case you are leveraging information across thousands of genes, and the resulting integration generally works quite well.
